lib/utils/getMathFunctionsRanges.js

```diff
@@ -1,12 +1,16 @@
 "use strict";
 
-const MATH_REGEXP = /calc\([^()]*(?:\(.*\))*[^()]*\)/gi;
+const MATH_FUNCTIONS = ["calc"];
+const MATH_FUNCTIONS_REGEXP = new RegExp(
+  `(?:${MATH_FUNCTIONS.join("|")})` + "\\([^()]*(?:\\(.*\\))*[^()]*\\)",
+  "gi"
+);
```

Comment:
Please don't use regexp for this cases, we have postcss-value-parser, regexp is slow and unsafe

Reply:
👌